bin/run_functions.js

```diff
@@ -29,6 +29,8 @@ module.exports.dispatcher = function (args) {
         'testsuite',
         'securitytest',
         'tags',
+        'callback',
+        'endpoint',
```

Comment:
**endpoint** should be called hostAndPort, the way it is in our API

Reply:
All occurrences of endpoint have been replaced with hostAndPort.